### PR TITLE
Added `--all-features` flag.

### LINKED ISSUE #1510: Workspaces and Features
Requested feature: Ability to deal with `--workspaces` and `Cargo.toml` features 
Use case: Potential customer code (see prost below) contain workspaces.
Link to relevant documentation (Rust reference, Nomicon, RFC): https://doc.rust-lang.org/cargo/reference/manifest.html

Test case: https://github.com/informalsystems/prost

### PR DESCRIPTION
**Pull request by YoshikiTakashima**
### Description of changes: 

Add the `--all-features` flag to cargo kani

### Resolved issues:

Resolves #1510 part 2

### Testing: 

* How is this change tested? tests/cargo-kani/feature-flag/

* Is this a refactor change? No

### Checklist
- [x] Each commit message has a non-empty body, explaining why the change was made
- [x] Methods or procedures are documented
- [x] Regression or unit tests are included, or existing tests cover the modified code
- [x] My PR is restricted to a single feature or bugfix

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 and MIT licenses.

### CODE REVIEW

File: kani-driver/src/args.rs [hunk 1]
```diff
@@ -114,6 +114,9 @@ pub struct KaniArgs {
     /// Kani will only compile the crate. No verification will be performed
     #[structopt(long, hidden_short_help(true))]
     pub only_codegen: bool,
+    /// Compile all features with kani.
```
Comment (celinval) [inline kani-driver/src/args.rs:117]: Can you please improve the doc? This is the cargo one:
> `--all-features`: Activates all features of all packages selected on the command-line.

Comment (YoshikiTakashima) [reply kani-driver/src/args.rs]: Addressed with 613160e1b55b08937351a19581ed56bbf6bb5874.

**Comment by YoshikiTakashima** [comment]
Should not take too long to review. Could someone check please?

**Review by tedinski — APPROVED** [review]

**Comment by YoshikiTakashima** [comment]
Thank you @tedinski

**Pull request merged**